README.md

```diff
@@ -42,6 +42,81 @@ func main() {
 	...
 ```
 
+## How to Start ##
+
+Line Messaging API use JSON to format data.
+```ParseRequest()``` will help you to parse the ```*http.Request``` content and return a slice of Event Object.
+
+```go
+    events := bot.ParseRequest(req)
```

Comment:
`ParseRequest` returns `[]*Event` and `error`.
So you have to add `err` in left side.